posix/consolefs.c

```diff
@@ -138,11 +139,68 @@ static int _consolefs_fcntl(oe_fd_t* file_, int cmd, uint64_t arg)
 {
     int ret = -1;
     file_t* file = _cast_file(file_);
+    void* argout = NULL;
+    uint64_t argsize = 0;
 
     if (!file)
         OE_RAISE_ERRNO(OE_EINVAL);
 
-    if (oe_posix_fcntl_ocall(&ret, file->host_fd, cmd, arg) != OE_OK)
+    switch (cmd)
+    {
+        case OE_F_GETFD:
+        case OE_F_SETFD:
+        case OE_F_GETFL:
+        case OE_F_SETFL:
+            break;
+
+        case OE_F_GETLK:
+        case OE_F_OFD_GETLK:
+            argsize = sizeof(struct oe_flock);
+            argout = (void*)arg;
+            break;
+
+        case OE_F_SETLKW:
+        case OE_F_SETLK:
+        {
+            void* srcp = (void*)arg;
```

Comment:
What is this code doing? It's copying arg to itself?  Isn't this case teh same for OE_F_GETLK?

Reply:
conver int64_t arg to a void *